src/instruments/src/EFB/Dashboard/Widgets/WeatherWidget.tsx

```diff
@@ -119,11 +120,11 @@ const WeatherWidget = (props: WeatherWidgetProps) => {
                                 <div className="ml-8">
                                     <IconCloud size={35} stroke={1.5} strokeLinejoin="miter" />
                                 </div>
-                                <input
-                                    className="text-left ml-4 border-none focus:outline-none text-2xl bg-transparent font-medium uppercase"
-                                    type="text"
+                                <SimpleInput
+                                    noLabel
+                                    className="text-center w-24 ml-4 text-2xl font-medium uppercase"
                                     placeholder={props.icao}
-                                    onChange={handleIcao}
+                                    onChange={(event) => handleIcao(event)}
```

Comment:
```suggestion
                                    onChange={(value) => handleIcao(value)}
```
There’s no event, it would make more sense to use value instead